integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonTableCompactor.scala

```diff
@@ -146,6 +147,10 @@ class CarbonTableCompactor(carbonLoadModel: CarbonLoadModel,
       maxSegmentColCardinality = null,
       maxSegmentColumnSchemaList = null,
       currentPartitions = partitions)
+    carbonLoadModel.setDateFormat(CarbonProperties.getInstance().
+      getProperty(CarbonCommonConstants.CARBON_DATE_FORMAT))
+    carbonLoadModel.setTimestampformat(CarbonProperties.getInstance().
+      getProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT))
```

Comment:
These could be table specific also not just system level, please ensure it  works for table level as well

Reply:
This was not required anymore so removed it.